Atomic/util.py

```diff
@@ -33,6 +33,7 @@
 BWRAP_OCI_PATH = "/usr/bin/bwrap-oci"
 RUNC_PATH = "/bin/runc"
 SKOPEO_PATH = "/usr/bin/skopeo"
+POLICY_FILENAME = "/etc/containers/policy.conf"
```

Comment:
`skopeo standalone-sign` doesn’t use `--policy`, and won’t.  I wouldn’t _entirely_ rule out `skopeo inspect`, and especially possible future commands, using it in the future—but that would be definitely discussed and coordinated, not a silent change.